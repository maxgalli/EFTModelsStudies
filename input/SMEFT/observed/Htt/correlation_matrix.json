{
    "r_smH_PTH_0_45": {
        "r_smH_PTH_0_45": 1.0,
        "r_smH_PTH_45_80": -0.06662553597682966,
        "r_smH_PTH_80_120": 0.08192361202201519,
        "r_smH_PTH_120_140": -0.08537149633375433,
        "r_smH_PTH_140_170": 0.026294595893651754,
        "r_smH_PTH_170_200": -0.01580896230083956,
        "r_smH_PTH_200_350": -0.10163146985175879,
        "r_smH_PTH_350_450": 0.07182742417709666,
        "r_smH_PTH_GT450": 0.003875784947936424
    },
    "r_smH_PTH_45_80": {
        "r_smH_PTH_0_45": -0.06662553597682966,
        "r_smH_PTH_45_80": 1.0,
        "r_smH_PTH_80_120": -0.6877971222819795,
        "r_smH_PTH_120_140": 0.3297473005690487,
        "r_smH_PTH_140_170": -0.03968858621814031,
        "r_smH_PTH_170_200": 0.021083099774895427,
        "r_smH_PTH_200_350": 0.25657033600146667,
        "r_smH_PTH_350_450": -0.6466267427366537,
        "r_smH_PTH_GT450": -0.0028567265255172616
    },
    "r_smH_PTH_80_120": {
        "r_smH_PTH_0_45": 0.08192361202201519,
        "r_smH_PTH_45_80": -0.6877971222819795,
        "r_smH_PTH_80_120": 1.0,
        "r_smH_PTH_120_140": -0.7492854960211521,
        "r_smH_PTH_140_170": 0.13518812192535976,
        "r_smH_PTH_170_200": -0.09203303760976075,
        "r_smH_PTH_200_350": -0.03852891367294114,
        "r_smH_PTH_350_450": 0.19826940311282035,
        "r_smH_PTH_GT450": -0.004453308037510122
    },
    "r_smH_PTH_120_140": {
        "r_smH_PTH_0_45": -0.08537149633375433,
        "r_smH_PTH_45_80": 0.3297473005690487,
        "r_smH_PTH_80_120": -0.7492854960211521,
        "r_smH_PTH_120_140": 1.0,
        "r_smH_PTH_140_170": -0.25640338569008486,
        "r_smH_PTH_170_200": 0.17017379636979316,
        "r_smH_PTH_200_350": 0.0377624246100107,
        "r_smH_PTH_350_450": -0.07126794204071492,
        "r_smH_PTH_GT450": 0.010294528733364165
    },
    "r_smH_PTH_140_170": {
        "r_smH_PTH_0_45": 0.026294595893651754,
        "r_smH_PTH_45_80": -0.03968858621814031,
        "r_smH_PTH_80_120": 0.13518812192535976,
        "r_smH_PTH_120_140": -0.25640338569008486,
        "r_smH_PTH_140_170": 1.0,
        "r_smH_PTH_170_200": 0.280956234512321,
        "r_smH_PTH_200_350": 0.009294963251660104,
        "r_smH_PTH_350_450": -0.01118261787347992,
        "r_smH_PTH_GT450": 0.33768456376578293
    },
    "r_smH_PTH_170_200": {
        "r_smH_PTH_0_45": -0.01580896230083956,
        "r_smH_PTH_45_80": 0.021083099774895427,
        "r_smH_PTH_80_120": -0.09203303760976075,
        "r_smH_PTH_120_140": 0.17017379636979316,
        "r_smH_PTH_140_170": 0.280956234512321,
        "r_smH_PTH_170_200": 1.0,
        "r_smH_PTH_200_350": -0.00280665829200077,
        "r_smH_PTH_350_450": 0.009239175413559104,
        "r_smH_PTH_GT450": -0.13346320295858874
    },
    "r_smH_PTH_200_350": {
        "r_smH_PTH_0_45": -0.10163146985175879,
        "r_smH_PTH_45_80": 0.25657033600146667,
        "r_smH_PTH_80_120": -0.03852891367294114,
        "r_smH_PTH_120_140": 0.0377624246100107,
        "r_smH_PTH_140_170": 0.009294963251660104,
        "r_smH_PTH_170_200": -0.00280665829200077,
        "r_smH_PTH_200_350": 1.0,
        "r_smH_PTH_350_450": -0.4505799033418916,
        "r_smH_PTH_GT450": -0.006526195926743245
    },
    "r_smH_PTH_350_450": {
        "r_smH_PTH_0_45": 0.07182742417709666,
        "r_smH_PTH_45_80": -0.6466267427366537,
        "r_smH_PTH_80_120": 0.19826940311282035,
        "r_smH_PTH_120_140": -0.07126794204071492,
        "r_smH_PTH_140_170": -0.01118261787347992,
        "r_smH_PTH_170_200": 0.009239175413559104,
        "r_smH_PTH_200_350": -0.4505799033418916,
        "r_smH_PTH_350_450": 1.0,
        "r_smH_PTH_GT450": 0.0072521517829419015
    },
    "r_smH_PTH_GT450": {
        "r_smH_PTH_0_45": 0.003875784947936424,
        "r_smH_PTH_45_80": -0.0028567265255172616,
        "r_smH_PTH_80_120": -0.004453308037510122,
        "r_smH_PTH_120_140": 0.010294528733364165,
        "r_smH_PTH_140_170": 0.33768456376578293,
        "r_smH_PTH_170_200": -0.13346320295858874,
        "r_smH_PTH_200_350": -0.006526195926743245,
        "r_smH_PTH_350_450": 0.0072521517829419015,
        "r_smH_PTH_GT450": 1.0
    }
}