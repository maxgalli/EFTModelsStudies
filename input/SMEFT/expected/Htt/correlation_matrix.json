{
    "r_smH_PTH_0_45": {
        "r_smH_PTH_0_45": 1.0,
        "r_smH_PTH_45_80": -0.11070431823148047,
        "r_smH_PTH_80_120": 0.04813159787078042,
        "r_smH_PTH_120_140": -0.028818084713647756,
        "r_smH_PTH_140_170": -0.009323241781270385,
        "r_smH_PTH_170_200": -0.011467645245415659,
        "r_smH_PTH_200_350": -0.33805723396726695,
        "r_smH_PTH_350_450": 0.22506207712046722,
        "r_smH_PTH_GT450": -0.008003428520297296
    },
    "r_smH_PTH_45_80": {
        "r_smH_PTH_0_45": -0.11070431823148047,
        "r_smH_PTH_45_80": 1.0,
        "r_smH_PTH_80_120": -0.8338954831706819,
        "r_smH_PTH_120_140": 0.5942586187634206,
        "r_smH_PTH_140_170": -0.2541959550019118,
        "r_smH_PTH_170_200": 0.08168107135133257,
        "r_smH_PTH_200_350": 0.352958330009273,
        "r_smH_PTH_350_450": -0.739088119030827,
        "r_smH_PTH_GT450": -0.03165216740746343
    },
    "r_smH_PTH_80_120": {
        "r_smH_PTH_0_45": 0.04813159787078042,
        "r_smH_PTH_45_80": -0.8338954831706819,
        "r_smH_PTH_80_120": 1.0,
        "r_smH_PTH_120_140": -0.8534138011565942,
        "r_smH_PTH_140_170": 0.44640478766980496,
        "r_smH_PTH_170_200": -0.13375837655026995,
        "r_smH_PTH_200_350": -0.16778024233171968,
        "r_smH_PTH_350_450": 0.4359424011861718,
        "r_smH_PTH_GT450": 0.060550867189618315
    },
    "r_smH_PTH_120_140": {
        "r_smH_PTH_0_45": -0.028818084713647756,
        "r_smH_PTH_45_80": 0.5942586187634206,
        "r_smH_PTH_80_120": -0.8534138011565942,
        "r_smH_PTH_120_140": 1.0,
        "r_smH_PTH_140_170": -0.5923879536019064,
        "r_smH_PTH_170_200": 0.19640327144470965,
        "r_smH_PTH_200_350": 0.1300172972818733,
        "r_smH_PTH_350_450": -0.3003310191659007,
        "r_smH_PTH_GT450": -0.07784056323945997
    },
    "r_smH_PTH_140_170": {
        "r_smH_PTH_0_45": -0.009323241781270385,
        "r_smH_PTH_45_80": -0.2541959550019118,
        "r_smH_PTH_80_120": 0.44640478766980496,
        "r_smH_PTH_120_140": -0.5923879536019064,
        "r_smH_PTH_140_170": 1.0,
        "r_smH_PTH_170_200": 0.026049035895227675,
        "r_smH_PTH_200_350": -0.029034209170240844,
        "r_smH_PTH_350_450": 0.1059370033902141,
        "r_smH_PTH_GT450": 0.25700002353684054
    },
    "r_smH_PTH_170_200": {
        "r_smH_PTH_0_45": -0.011467645245415659,
        "r_smH_PTH_45_80": 0.08168107135133257,
        "r_smH_PTH_80_120": -0.13375837655026995,
        "r_smH_PTH_120_140": 0.19640327144470965,
        "r_smH_PTH_140_170": 0.026049035895227675,
        "r_smH_PTH_170_200": 1.0,
        "r_smH_PTH_200_350": 0.014172373315443933,
        "r_smH_PTH_350_450": -0.030748057380654607,
        "r_smH_PTH_GT450": -0.2725367773493211
    },
    "r_smH_PTH_200_350": {
        "r_smH_PTH_0_45": -0.33805723396726695,
        "r_smH_PTH_45_80": 0.352958330009273,
        "r_smH_PTH_80_120": -0.16778024233171968,
        "r_smH_PTH_120_140": 0.1300172972818733,
        "r_smH_PTH_140_170": -0.029034209170240844,
        "r_smH_PTH_170_200": 0.014172373315443933,
        "r_smH_PTH_200_350": 1.0,
        "r_smH_PTH_350_450": -0.5322876881592096,
        "r_smH_PTH_GT450": -0.004466735452110171
    },
    "r_smH_PTH_350_450": {
        "r_smH_PTH_0_45": 0.22506207712046722,
        "r_smH_PTH_45_80": -0.739088119030827,
        "r_smH_PTH_80_120": 0.4359424011861718,
        "r_smH_PTH_120_140": -0.3003310191659007,
        "r_smH_PTH_140_170": 0.1059370033902141,
        "r_smH_PTH_170_200": -0.030748057380654607,
        "r_smH_PTH_200_350": -0.5322876881592096,
        "r_smH_PTH_350_450": 1.0,
        "r_smH_PTH_GT450": 0.013593332736269724
    },
    "r_smH_PTH_GT450": {
        "r_smH_PTH_0_45": -0.008003428520297296,
        "r_smH_PTH_45_80": -0.03165216740746343,
        "r_smH_PTH_80_120": 0.060550867189618315,
        "r_smH_PTH_120_140": -0.07784056323945997,
        "r_smH_PTH_140_170": 0.25700002353684054,
        "r_smH_PTH_170_200": -0.2725367773493211,
        "r_smH_PTH_200_350": -0.004466735452110171,
        "r_smH_PTH_350_450": 0.013593332736269724,
        "r_smH_PTH_GT450": 1.0
    }
}